trl/trainer/ppo_trainer.py

```diff
@@ -1382,3 +1390,35 @@ def _show_tokens(self, tokens, masks):
                 text.append(self.tokenizer.decode(token.item()), style="black on cyan3")
                 text.append(" ")
         print(text)
+
+    def _prepare_deepspeed_zero3(self, model):
+        # Adapted from accelerate: https://github.com/huggingface/accelerate/blob/739b135f8367becb67ffaada12fe76e3aa60fefd/src/accelerate/accelerator.py#L1473
+        # TODO: figure out if any other parameters are needed for inference
```

Comment:
The kwargs below are a best guess for what's minimally needed - we can tune them later if needed IMO